src/client/src/core/util/validation/rules.ts

```diff
@@ -361,7 +361,8 @@ export const validFileTypes = function (fileTypes: string[]) {
 export const validEmailFormat = function (value: string): boolean | string {
   return (
     !value ||
-    /^(([^<>()[\]\\.,;:\s@#\-"]+(\.[^<>()[\]\\.,;:\s@#\-"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/.test(
+    // eslint-disable-next-line no-useless-escape
+    /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/.test(
```

Comment:
seems better if we revert this;

https://github.com/symfony/symfony/blob/6.2/src/Symfony/Component/Validator/Constraints/EmailValidator.php#L29
https://www.w3.org/TR/2012/WD-html-markup-20121025/input.email.html#form.data.emailaddress_xref2

Reply:
Reverted